torchani/utils.py

```diff
@@ -142,8 +143,31 @@ class EnergyShifter(torch.nn.Module):
 
     def __init__(self, self_energies):
         super(EnergyShifter, self).__init__()
-        self_energies = torch.tensor(self_energies, dtype=torch.double)
+
+        if self_energies is not None:
+            self_energies = torch.tensor(self_energies, dtype=torch.double)
+
         self.register_buffer('self_energies', self_energies)
+        self.present_species = present_species
+
+    def sae_from_dataset(self, atomic_properties, properties):
```

Comment:
Could you make this a `@staticmethod`? No member function is used in this function.

Reply:
Fixed in dc85d3c